apps/explorer/src/components/orders/FilledProgress/index.tsx

```diff
@@ -206,7 +206,7 @@ export function FilledProgress(props: Props): JSX.Element {
   const mainSymbol = mainToken ? safeTokenName(mainToken) : mainAddress
   const swappedSymbol = swappedToken ? safeTokenName(swappedToken) : swappedAddress
   // In case the token object is empty, display the raw amount (`decimals || 0` part)
-  const formattedPercentage = filledPercentage.toNumber() * 100
+  const formattedPercentage = filledPercentage.times(100)
```

Comment:
Second issue.

BigNumber converted to Number then multiplied by 100, losing the precision from BigNumber.